CHANGELOG.md

```diff
@@ -33,7 +33,7 @@ Breaking changes:
   * CMake options `WITH_OTLP_HTTP_SSL_PREVIEW`
     and `WITH_OTLP_HTTP_SSL_TLS_PREVIEW` are removed.
     Building opentelemetry-cpp without SSL support is no longer possible.
-
+* [#240] Change default OTLP Http Exporter content type from kJson to kBInary
```

Comment:
The PR number is 2480

What is the rationale for this change ?

Reply:
Found it, the proper link is:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specify-protocol

The default is http/protobuf, which means binary.

Reply:
> The PR number is 2480
> 
> What is the rationale for this change ?

Fixed PR number.

Reply:
This change is also a 3x to 10x performance improvement in my tests.